docs/consent/template-architecture.md

```diff
@@ -0,0 +1,51 @@
+## Overview
```

Comment:
Missing the page title - this is removed from the rendered version and added as a proper H1, also used for the title tag:
```suggestion
# Template Architecture

## Overview
```